Dockerfile

```diff
@@ -1,37 +1,39 @@
-FROM phusion/baseimage:0.9.19
+FROM phusion/baseimage:0.9.22
```

Comment:
If a slimmer image works, I would go with that as it's always easier to work with lightweight containers. Seems like they've slimmed down the official ubuntu image to around ~125MB so should be okay